README.md

```diff
@@ -283,6 +68,14 @@ We are eager to explore integrations with other computational backends such as [
 
 Have questions, suggestions, or comments? You can come hang out with us by joining the [CZI Science Slack](https://join-cellxgene-users.herokuapp.com/) and posting in the `#cellxgene-users` channel. As mentioned above, please submit any feature requests or bugs as [Github issues](https://github.com/chanzuckerberg/cellxgene/issues). We'd love to hear from you!
 
+## core team
+
+- Colin Megill, frontend & product design
+- Charlotte Weaver, backend engineer
```

Comment:
can I be full stack engineer or software engineer instead?

Reply:
yup